device/transport/amqp/src/amqp_device_method_client.ts

```diff
@@ -104,6 +104,8 @@ export class AmqpDeviceMethodClient extends EventEmitter {
                     this._fsm.transition('detaching', attachCallback, err);
                   } else {
                     this._senderLink = senderLink;
+                    const acceptProperty = 'autoaccept';
```

Comment:
Is this line necessary or can you just set this string as the key?

Reply:
Fixed.